### DISCUSSION TITLE
Plugin question?

### DISCUSSION THREAD

**Discussion by ShepardPl** [TownyAdvanced/Towny#5358] [Q&A] [answered]
Hello, this plugin is really good, primarily because of its capabilities and huge content. In order to understand the plugin, the guides helped me, but here's what I noticed - they talk about the commands / toce, / tocebank, / townforms, / typetown, / townreligions. But in the plugin itself, these commands and functions associated with them are also absent.
Are these some additional plugins for Towny?

**Comment by LlmDl** [comment, marked as answer]
None of those commands are built into Towny. I'm not sure which plugins add them. You're probably not reading guides hosted on our wiki.
